spec/rails_helper.rb

```diff
@@ -14,6 +16,18 @@
   config.include FactoryGirl::Syntax::Methods
 end
 
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+  
+  config.shared_context_metadata_behavior = :apply_to_host_groups
+end
```

Comment:
Yeah we got it from the cooper documentation. @tochman Can you have a look at it?